tools/WinObjC.Language/WinObjC.Language.Packaging.targets

```diff
@@ -3,11 +3,12 @@
 
   <Target Name="AddBuiltOutput" BeforeTargets="GetPackageContents" Condition="'$(SkipGetPackageContentsForOtherPlatforms)' != 'true'">
       <ItemGroup>
-          <!-- Inlcude *all* of /include/  and /include_next/ folder.  -->
-          <_IncludeFiles Include="$(MSBuildThisFileDirectory)..\include\**\*.*"/>
+          <!-- Include *all* of /include/, /include/xplat/ and /include_next/ folder.  -->
+          <_IncludeFiles Include="$(MSBuildThisFileDirectory)..\include\**\*.*" Exclude="$(MSBuildThisFileDirectory)..\include\xplat\**\*.*"/>
+          <_IncludeFiles Include="$(MSBuildThisFileDirectory)..\include\xplat\**\*.*"/>
```

Comment:
Most projects include Starboard.h and that includes Starboard\SmartTypes.h.
So, we should just have to change that to xplat\Starboard\smarttypes.h

And for the files that directly include <Starboard/SmartTypes.h>we change them to xplat/Starboard/SmartTypes.h



---
In reply to: [111036207](https://github.com/Microsoft/WinObjC/pull/2493#discussion_r111036207) [](ancestors = 111036207)